app.py

```diff
@@ -36,6 +36,7 @@ def index():
         email = request.form['email']
         TRAVIS_TAG = request.form['TRAVIS_TAG']
         event_url = request.form['event_url']
+        GENERATOR_ = request.form['GENERATOR_']
```

Comment:
Cool.
To get all variables, you should be able to do something like

    variables = {}
    for name, value in request.form.items():
        if name.startswith("GENERATOR_"):
            variables[name] = value